### PR TITLE
Allow top-level return in script parsing. Fixes issue 8509 in master,…

### PR DESCRIPTION
**Pull request by mwijngaard**
… like PR 8510 fixed it for 'next'

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Please accept CLI

File: test/fixtures/b.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 module.exports = function b() {
 	return "This is b";
-};
\ No newline at end of file
+};
+
+// Test CJS top-level return
+return;
```
Comment (evilebottnawi) [inline test/fixtures/b.js]: Can we add new test instead modify current?

Comment (mwijngaard) [reply test/fixtures/b.js]: Thanks for reviewing. The idea is for this to be exactly the same change as https://github.com/webpack/webpack/pull/8510 , which was already merged into 'next'. If you really prefer the test to be split it up I can to that, but it seems to me that that does not benefit understandability when comparing the branches.

Comment (evilebottnawi) [reply test/fixtures/b.js]: :+1:

**Comment by sokra** [comment]
Thanks

**Pull request merged**